apps/metric_vnode/src/metric_vnode.erl

```diff
@@ -132,40 +132,40 @@ handle_command({repair, Bucket, Metric, Time, Value}, _Sender,
         {true, State1} ->
             State2 =
                 case ets:lookup(T, {Bucket, Metric}) of
-                    %% If we repear ona a place before the metric, well just
+                    %% If we repair on a place before the metric, we will just
                     %% write it!
                     [{{Bucket, Metric}, Start, _Size, _Time, _Array}]
                       when Time + Count < Start ->
                         metric_io:write(State#state.io, Bucket, Metric, Time,
                                         Value),
                         State1;
-                    %% The data is entirely behind the cache, so we flush the
-                    %% cache and use the repair request as new cache
+                    %% The data is entirely ahead of the cache, so we flush the
+                    %% cache and use the repair request as new the cache.
                     [{{Bucket, Metric}, Start, Size, _Time, Array}]
-                      when Start + Size > Time ->
+                      when Time > Start + Size ->
```

Comment:
This might seem a bit very picky but I'd prefer this to read 
```erlang
when  Start + Size < Time
```

mostly because the visualization is 'further right' and we draw time fron the left so the larger value on the right might be easier to read.